sourced/ml/algorithms/uast_ids_to_bag.py

```diff
@@ -70,3 +72,23 @@ def __init__(self, token2index=None, token_parser=None):
         """
         token_parser = TokenParser() if token_parser is None else token_parser
         super().__init__(token2index, token_parser)
+
+    def __call__(self, uast):
+        """
+        Converts a UAST to a weighed bag-of-identifiers. The weights are identifiers frequencies.
```

Comment:
Please add a big visible FIXME for libuast override

Reply:
I will put `FIXME bblfsh.filter` - I'm not sure if it's `bblfsh-client` problem or `libuast`. Anyway I submitted issue to `bblfsh-client`:  https://github.com/bblfsh/client-python/issues/92